parser_test.go

```diff
@@ -190,7 +190,7 @@ func TestParser_ParseTransactions(t *testing.T) {
 			height:  "845259",
 			results: expectedResults{
 				totalTraces:  31,
-				totalAddress: 3,
+				totalAddress: 2,
```

Comment:
I tried with version 1.28 and it ALSO gives this error. We can discuss this in a meeting